src/main/groovy/com/github/eirnym/js2p/JsonSchemaPlugin.groovy

```diff
@@ -73,6 +73,7 @@ class JsonSchemaPlugin implements Plugin<Project> {
                 }
 
                 JsonSchemaExtension config = project.extensions.getByType(JsonSchemaExtension)
+                config.includeGeneratedAnnotation = false  // Disable generated annotation for Android users by default
```

Comment:
What's the benefit ?

Reply:
AGP 7+ requires Java 11 according to the [article](https://android-developers.googleblog.com/2020/12/announcing-android-gradle-plugin.html?m=1), so we need eventually test it as well

For java9+ _Gradle's Java_ (this is not about target java level) jsonschema2pojo library sets Generated annotation as javax.annoyations.**processing**.Generated, and I haven't found any library for this annotation. 

So I just disable this annotation for Android builds. Should I check _Gradle's java_ version aim on java 9+ only with this check?

Reply:
It it makes sense: yes

Reply:
Keep in mind, every time someone ask you something, it's indicate a missing explanation

Reply:
I'll add this in README file

Reply:
It's just my personal opinion: nobody will read the README or will understand the link to this line.
As least for me, direct comment this line would help most

Reply:
I agree with your opinion. I'll make an explicit issue and PR for that.